core/io/image_loader.cpp

```diff
@@ -62,7 +62,7 @@ Error ImageLoader::load_image(String p_file, Ref<Image> p_image, FileAccess *p_c
 
 		if (!loader[i]->recognize(extension))
 			continue;
-		Error err = loader[i]->load_image(p_image, f, p_force_linear);
+		Error err = loader[i]->load_image(p_image, f, p_force_linear, p_scale);
```

Comment:
@toger5 but why would you like to import a non vector texture as a bigger texture, in any case you can resize the sprite later on, i see no point in having such import option on sprites, the scale is for rasterizing purposes, also we would have to update all the image loaders to support a feature that does not make sense on raster images.